### DISCUSSION TITLE
Add option to view feed by date

### DISCUSSION THREAD

**Discussion by GrimLink** [community/community#13180] [Other Feature Feedback, Questions, & Ideas]
I find the idea interesting but one thing do dislike is the way how stuff is ordered.

I like my feed ordered by date,
not based on some random order based on my interests.

It would be nice if there was a option to pick the feed order.

**Comment by volksec** [comment]
UP!

**Comment by jfullstackdev** [comment]
this might be able to clean your dashboard

on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
